libfoundation/src/system-file-posix.cpp

```diff
@@ -364,8 +364,6 @@ __MCSFileSetContents (MCStringRef p_native_path,
 
 	if (0 != rename (*t_temp_path_sys, *t_path_sys))
 	{
-		int t_save_errno = errno;
-
```

Comment:
@montegoulding I honestly believe that Xcode's indentation is pretty buggy.  The diff shows that the `unlink()` line started off indented with tabs, but your Xcode has somehow managed to replace them with spaces.

I have my Emacs configured to indent with tabs and align with spaces, but unfortunately I don't think there are any other editors that will do that correctly.

For a while I've been wondering about having a flag day where we run a tool like [uncrustify](http://uncrustify.sourceforge.net/) over all our C++ and C code (excluding 3rd party) and add a rule to Vulcan to fail the build for "incorrectly"-indented code.  It would certainly solve all the "misleading indentation" rules I get from GCC6 due to inconsistent indentation...

In any case this isn't something that blocks the patch, it's just something I was curious about.  Feel free to either add the missing `errno = t_save_errno` line, or mark this PR as reviewed by me. 👍